nbm-maven-plugin/src/it/custommoduleoverride/application/pom.xml

```diff
@@ -0,0 +1,133 @@
+            <artifactId>platform</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.cluster</groupId>
+            <artifactId>ide</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.netbeans.modules</groupId>
+                    <artifactId>org-netbeans-libs-jaxb</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency> 
+        <dependency>
+            <groupId>org.netbeans.cluster</groupId>
+            <artifactId>harness</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+        </dependency>   
+        <dependency>
+            <groupId>org.netbeans.cluster</groupId>
+            <artifactId>java</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+            
+        </dependency>  
+        <dependency>
+            <groupId>org.netbeans.cluster</groupId>
+            <artifactId>extide</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+        </dependency> 
+        <dependency>
+            <groupId>org.netbeans.cluster</groupId>
+            <artifactId>websvccommon</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.cluster</groupId>
+            <artifactId>webcommon</artifactId>
+            <version>${netbeans.version}</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>moduleoverride-branding</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.netbeans.api</groupId>
+            <artifactId>org-netbeans-modules-nbjunit</artifactId>
+            <version>${netbeans.version}</version>
+            <scope>test</scope>
+
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>moduleoverride-jaxbwrapper</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.netbeans.utilities</groupId>
+                <artifactId>nbm-maven-plugin</artifactId>
+                <configuration>
+                    <verifyRuntime>fail</verifyRuntime>
+                    <additionalArguments>-J--add-opens=java.base/java.net=ALL-UNNAMED -J--add-opens=java.desktop/javax.swing.text=ALL-UNNAMED -J--add-opens=java.prefs/java.util.prefs=ALL-UNNAMED</additionalArguments>
```

Comment:
use `jpms-flags` property of Line 33? (or remove property since it seems to be unused otherwise)

Reply:
it miss the -J if I recall well